mandatory/1-create-functions.js

```diff
@@ -3,16 +3,14 @@ Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the first five elements of the passed array.
 */
-function first5() {
-}
+function first5() {}
```

Comment:
Please can we get rid of this function too whilst we're here?

Reply:
I'm not sure I follow - are you asking to remove this (sub-)exercise? Or to remove the stubbed function?

If you're asking for the latter, I think we do need it as we call this specific named function in the tests, so getting trainees to write it themselves risks misnaming, which could lead to confusing errors. I guess I would be fine with going down that route, but that feels like a policy decision that should be taken across the board.